adele-init/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>project-adele</artifactId>
+        <groupId>com.microservicesteam</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>adele-init</artifactId>
```

Comment:
I created the `adele-init` module, because I wanted to move everything here, which is not related strictly to the production code, but needed for local testing. Although `TestPublisherController` and `TestRandomBookingEventScheduler` were thrown out in the meantime, I think it's still a good idea to have these kind of classes in their own module, so they can be thrown out / removed from the build easily.